docs/content/contributing/bicep/bicep-contribution-flow/child-module-publishing.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Child Module Publishing
+description: How to set up a child module for publishing so that it can be directly referenced from the Bicep public registry
```

Comment:
It's
```suggestion
description: How to set up a child module for publishing so that it can be directly referenced from the Public Bicep Registry
```
right?